### PR TITLE
Supervised Learning with Backpropogation Algorithm

### PR DESCRIPTION
**Pull request by QuantumAna**
Artificial neural networks (ANNs) are powerful tools for machine learning with
applications in many areas including speech recognition, image classification,
medical diagnosis, and spam filtering. The Backpropagation algorithm which
learns the weights for a multilayer network, given a network with a fixed set of
units and interconnections. It employs gradient descent to attempt to minimize
the squared error between the network output values and target values for these
outputs.

### CODE REVIEW

**Comment by lazyprogrammer** [comment]
This repository already contains code for backpropagation.
